css/properties/hyphens.json

```diff
@@ -461,7 +461,7 @@
                 "version_added": "5.1"
               },
               "safari_ios": {
-                "version_added": false
+                "version_added": "6"
```

Comment:
"5" would be consistent with rest, wouldn't it?

Reply:
I guess so, yeah!

```suggestion
                "version_added": "5"
```